scripts/regen-cabal.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -eux
+
+stack2cabal_ver="1.0.11"
+
+SOURCE="$0"
+SCRIPT_DIR="$(dirname "$(realpath "${SOURCE}")")"
+
+cd "${SCRIPT_DIR}"/..
+
+mkdir -p .tmp/bin
+export PATH="$(realpath .tmp/bin):$PATH"
```

Comment:
So we currently always prefer a local copy of the binary, if there is one (over a system one). I think that's fair.